### DISCUSSION TITLE
后台左侧Sidebar在特定分辨率下无法呈现完全

### DISCUSSION THREAD

**Discussion by n0ah37** [Baiyuetribe/kamiFaka#36] [Bug反馈]
![image](https://user-images.githubusercontent.com/71708018/106380009-892f2f00-63ea-11eb-98cf-a4b024abfcc0.png)
如图所示
将网站目录内/dist/css/styles.7e54a98a.css内第3447行
``
.main-sidebar {
    box-shadow: 0 4px 8px rgba(0, 0, 0, .03);
    position: fixed;
    top: 0;
    height: 100%;
    width: 250px;
    background-color: #fff;
    z-index: 880;
    left: 0
}
 ``
修改成
``
.main-sidebar {
    box-shadow: 0 4px 8px rgba(0, 0, 0, .03);
    position: fixed;
    top: 0;
    height: 100%;
    width: 250px;
    background-color: #fff;
    z-index: 880;
    left: 0;
    overflow: auto
}
 ``
即可解决
![image](https://user-images.githubusercontent.com/71708018/106380146-4ae63f80-63eb-11eb-8879-45e12aa0f8e3.png)

**Comment by Baiyuetribe** [comment]
感谢反馈，该问题终于修复了
